sdk/ml/azure-ai-ml/azure/ai/ml/_exception_helper.py

```diff
@@ -143,47 +143,57 @@ def format_details_section(
     return error_types, details
 
 
-def format_errors_and_resolutions_sections(
-    entity_type: str, error_types: Dict[str, bool], cli: bool
-) -> Tuple[str, str]:
+def format_errors_and_resolutions_sections(error_types: Dict[str, bool], cli: bool) -> Tuple[str, str]:
     """Builds strings for details of the error message template's Errors and Resolutions sections."""
 
     resolutions = ""
     errors = ""
     count = 1
 
     if error_types[ValidationErrorType.INVALID_VALUE]:
-        errors += f"\n{count}) One or more fields are invalid"
+        errors += (
+            f"\n{count}) The yaml file you provided has one or more values" f" (for required fields) which are invalid:"
+        )
         resolutions += (
-            f"\n{count}) Double-check that all specified parameters are of the correct types and formats "
-            f"prescribed by the {entity_type} schema."
+            f"\n{count}) If the type is invalid, change it to the correct type. "
+            f"If value is invalid, choose a value from the expected range."
         )
         count += 1
     if error_types[ValidationErrorType.UNKNOWN_FIELD]:
-        errors += f"\n{count}) A least one unrecognized parameter is specified"
-        resolutions += f"\n{count}) Remove any parameters not prescribed by the {entity_type} schema."
+        errors += f"\n{count}) The yaml file you provided has one or more unrecognized parameters. "
+        resolutions += f"\n{count}) Remove the unrecognized parameter specified above."
         count += 1
     if error_types[ValidationErrorType.MISSING_FIELD]:
-        errors += f"\n{count}) At least one required parameter is missing"
-        resolutions += f"\n{count}) Ensure all parameters required by the {entity_type} schema are specified."
+        errors += f"\n{count}) The yaml file provided is missing one or more required parameters."
+        resolutions += (
+            f"\n{count}) Add the missing parameters with values of the correct type. "
+            f"To know what type of value to assign to any required parameter, "
```

Comment:
I'm assuming this is still in progress, but is there a point to this being an f string? Looks like it doesn't reference any variable. Presumably this would be the spot for reference information to the user.

Reply:
Yeah, there was no point in having f strings for lines like 170. I went back and removed unnecessary f strings. The error handler includes relevant references to the entity type and is appended later on.